tests/tasks/test_api_create_task.py

```diff
@@ -61,8 +59,7 @@ def test_create_task_api_token_expired(self):
     def test_create_task_api_resource_non_auth(self):
         expected_response = messages.AUTHORISATION_TOKEN_IS_MISSING
         actual_response = self.client.post(
-            f"/mentorship_relation/{self.mentorship_relation_w_second_user.id}"
-            "/task",
+            f"/mentorship_relation/{self.mentorship_relation_w_second_user.id}" "/task",
```

Comment:
Can you please remove the empty space in the string, like this and apply it to other examples here in the code?
```suggestion
            f"/mentorship_relation/{self.mentorship_relation_w_second_user.id}/task",
```